sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -345,19 +345,30 @@ def _try_get_data_binding(_input_output_data) -> Union[str, None]:
                     return None
                 return _input_output_data._data_binding()
 
+            def _is_group_input(_input_obj) -> bool:
+                # exclude group input for now as it only comes from pipeline input now
+                # TODO: update validate init/finalize logic after supporting group as output
```

Comment:
Please create a task under 1.4.0 to track this.

Reply:
mmm.. we can also reuse the exiting bug

Reply:
I have updated implementation and maybe this can be solved in this PR